src/lib/modernExtend.ts

```diff
@@ -1713,9 +1713,9 @@ export function electricityMeter(args?: ElectricityMeterArgs): ModernExtend {
         },
         seMetering: {
             // Report change with every 5W change
-            power: {attribute: 'instantaneousDemand', divisor: 'divisor', multiplier: 'multiplier', forced: args.power, change: 5},
+            power: {attribute: 'instantaneousDemand', divisor: 'divisor', multiplier: 'multiplier', forced: args.power, change: 0.005, min: 'min'},
             // Report change with every 0.1kWh change
-            energy: {attribute: 'currentSummDelivered', divisor: 'divisor', multiplier: 'multiplier', forced: args.energy, change: 0.1},
+            energy: {attribute: 'currentSummDelivered', divisor: 'divisor', multiplier: 'multiplier', forced: args.energy, change: 0.1, min: 'min'},
```

Comment:
Why was the `change` changed here? (also for the line above)

Reply:
I didn't change it for energy/currentSummDelivered. `change` for energy is still 0.1 as it has always been. I just added the property `min` at the end. 

I did however change the value for power/instantaneousDemand from 5kWh to 0.005kWh as mentioned in this issue https://github.com/Koenkk/zigbee-herdsman-converters/issues/7917#issuecomment-2323316989. With  a `change` value of 5 for power I get a front end reporting Min Rep Change value of 50,000 (because the divisor for my E13-A21 is 10,000).

Reply:
I see, thanks!